tasks/sep/006-0-sc-changes/README.md

```diff
@@ -0,0 +1,31 @@
+# Sepolia Guardian Changes - Security Council Runbook
+
+Status: READY TO SIGN
+
+## Objective
+
+This is the sepolia playbook for:
+
+1. Finalizing the configuration of a newly deployed Security Council Safe by enabling the LivenessModule on it.
+1. Removing the extra deployer key owner.
+
+On Mainnet the threshold of the Security Council will also need to be increased, but that is not necessary here.
```

Comment:
Let's also link to any relevant docs and justifications for this runbook (i.e. same links we'd want for the mainnet playbook). Similar to how in other playbooks we have checkboxes saying "gov proposal approved", "not vetoed", etc